syntax = "proto3";

package vonk_fence;

message FirmwareIn {
    optional uint32 ping = 1;
    uint32 flash_millis = 2;
    optional DataRequest data_request = 3;
}

message DataRequest {
    bool volume = 1;
    bool region_of_interest = 2;
}
